_layouts/about.html

```diff
@@ -12,13 +12,19 @@
 </div>
 <div id="staff-list">
     {% for staff in page.other-staff %}
-        <img src={{staff.image-path}} alt={{staff.name}}></img>
-        {% if staff.bio-url %}
-            <h3><a href="{{staff.bio-url}}">{{staff.name}}</a></h2>
-        {% else %}
-            <h3>{{staff.name}}</h2>
-        {% endif %}
-        <p>{{staff.title}}</p>
+    <div class="row">
+        <div class="people-img col-sm-7">
+            <img class="" src={{staff.image-path}} alt={{staff.name}}></img>
```

Comment:
Also, I noticed a bug with the templating. If spaces exist in HTML for alt text, it needs to wrapped in strings. `<img src="{{staff.image-path}}" alt="{{staff.name}}">` would fix the bug.